### PR TITLE
 dotCMS/core#18021 [Content] : Container not behaving correctly when reaching max contents

### PR DESCRIPTION
**Pull request by alfredo-dotcms**
this covers only the disabling of the menu actions when contentlets limit has been reached

### CODE REVIEW

**Comment by alfredo-dotcms** [comment]
the D&D is WIP

**Comment by wezell** [comment]
@alfredo-dotcms please make sure this respects languages - meaning if the container is set for 3 contentlets, each language should be able to have 3 contentlets.

File: src/app/portlets/dot-edit-page/content/services/html/dot-edit-content-toolbar-html.service.ts [hunk 1]
```diff
@@ -202,14 +225,39 @@ export class DotEditContentToolbarHtmlService {
                             uuid: container.dataset.dotUuid
                         },
                         disabled: isDisabledFormAdd,
-                        tooltip: isDisabledFormAdd
-                            ? this.dotMessageService.get('dot.common.license.enterprise.only.error')
-                            : ''
+                        tooltip: this.getTooltipErrorMessage(
+                            isDisabledFormAdd,
+                            maxContentletsLimitReached
+                        )
```
Comment (freddyucv) [inline src/app/portlets/dot-edit-page/content/services/html/dot-edit-content-toolbar-html.service.ts]: I think all this code is very weird and not readable enoght, maybe it is better do something like:

```suggestion
                     let validationError: ValidationError = getContentletValidationError(item);
                     ...
                     tooltip: validationError === ValidationErrorMessage(validationError)
                     ...
                     getContentletValidationError(item): ValidationError {
                          if (item === 'form' && !this.isEnterpriseLicense) {
                              return ValidationError.EnterpriseLicenseError;
                          } else if (this.isMaxContentletsLimitReached(container)) {
                              return ValidationError.MaxContentletsLimitReacheError;
                          }
                     }
```

where ValidationError is a enum

**Comment by alfredo-dotcms** [comment]
@freddyucv feedback uploaded!

**Pull request merged**